users/templates/user_site/home.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+
+{% block head-content %}
+    <meta charset="utf-8"/>
+    <title>stocks</title>
+    <meta content="stocks" property="og:title"/>
+    <meta content="stocks" property="twitter:title"/>
+    <meta content="width=device-width, initial-scale=1" name="viewport"/>
+
+    {% load static %}
+    <link href="{% static 'user_site/home.css' %}" rel="stylesheet">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://cdn.jsdelivr.net/npm/sweetalert2@10"></script>
+    <script src="https://cdn.lordicon.com//libs/frhvbuzj/lord-icon-2.0.2.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/4.0.2/bootstrap-material-design.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.6/css/bootstrap.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" type="text/javascript"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+    <style>
+        #overlay {
+            position: fixed; /* Sit on top of the page content */
+            display: none; /* Hidden by default */
+            width: 100%; /* Full width (cover the whole page) */
+            height: 100%; /* Full height (cover the whole page) */
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            background-color: rgba(0,0,0,0.5); /* Black background with opacity */
+            z-index: 2; /* Specify a stack order in case you're using a different order for other elements */
+            cursor: pointer; /* Add a pointer on hover */
+        }
+    </style>
+
+{% endblock %}
+
+{% block body-content %}
+    <div id="overlay"></div>
+    <div class="table-title">
+        <h1 style="font-family: 'Bodoni Moda', serif; font-size: 4.8em;">{{ month_name }}:</h1>
+    </div>
+    <div style="display: inline;">
+        <div class="table-icon" style="float:left; padding-left: 6%;">
+            <a href="/admin" style="color: black;">
```

Comment:
why does the plus icon refrence the admin page ? do we want anyone to be able to access the admin page?

Reply:
The bottoms actions are now for testing proposes only,
In a later stage the ➕  sign will be used to add events,

I have change it to go to #event_change and the relevant PR will change it accordingly